consensus/poet/core/tests/test_consensus/test_poet_block_publisher.py

```diff
@@ -674,3 +674,233 @@ def test_block_publisher_success_case(
         self.assertTrue(
             block_publisher.initialize_block(
                 block_header=mock_block.header))
+
+    @patch('sawtooth_poet.poet_consensus.wait_timer.WaitTimer')
```

Comment:
Please change `@patch` to `@mock.patch` to be consistent with other patches.  You'll also want to remove the `from unittest.mock import patch` at the top of file.